xml/net_samba.xml

```diff
@@ -148,7 +148,15 @@
    </varlistentry>
   </variablelist>
  </sect1>
- <sect1 xml:id="sec.samba.install">
+ <sect1 xml:id="sec.samba.server-sled" os="sled">
+  <title>Installing a Samba Server</title>
+  <para>
+   Installing a Samba server is not supported on &sled;.
+   For information about installing and configuring a Samba server, see
+   &admin; for &sls;.
```

Comment:
'see &admin;' -> 'see the &admin;' ?

Reply:
Ignored for now. If we omit "the" here, it is easier to change the reference later.